client/src/EntryList.jsx

```diff
@@ -0,0 +1,45 @@
+import EntryItem from './EntryItem';
+
+export default function EntryList({ data, onClick }) {
+  return (
+    <>
+      <div className="container" data-view="entries">
+        <div className="row">
+          <div className="column-full d-flex justify-between align-center">
+            <h1>Entries</h1>
+            <h3>
+              <a
+                onClick={onClick}
```

Comment:
example of changing view re: `App.jsx`: 

`onClick={() => setView("entry-form")}`